### PR TITLE
Fix generator to work with node 0.12.0 and updated out of date dependenci...

### PR DESCRIPTION
**Pull request by nigampatel**
I have fix out of date dependencies and made some updates where generator can work with new node version 0.12

### CODE REVIEW

File: controller/index.js [hunk 1]
```diff
@@ -29,10 +29,24 @@ var util = require('util'),
...
     yeoman.generators.Base.apply(this, arguments);
+    var cwd = this.env.cwd,
+        filePath = path.join(cwd, 'package.json'),
+        packageTemplate = null;
 
     krakenutil.update();
 
-    this.hasTemplates = (args[1]) ? true : false;
+    this.hasTemplates = (args[1]) ? true : (
+        fs.existsSync(filePath) ? (
```
Comment (pvenkatakrishnan) [inline controller/index.js]: looks like you have forgotten to require `fs` causing the build to break for your pr. Can you fix it ?

Comment (aredridel) [inline controller/index.js]: Looks like that's fixed!

**Comment by aredridel** [comment]
Rebased and fixed by #168

**Comment by aredridel** [comment]
Thank you very much for this contribution!

**Pull request closed without merging**